### DISCUSSION TITLE
with_nodelay on prebound server

### DISCUSSION THREAD

**Discussion by joshtriplett** [trillium-rs/trillium#386] [General]
I'd like to use trillium with a prebound server, so that I can listen on 443 and then drop privileges.

Trillium rejects `with_nodelay` if it has a prebound server.

However, as far as I can tell, without this restriction, it would work perfectly, just as it does with a listener from `LISTEN_FD`. Either way, trillium would get a Transport from the listener, call set_nodelay on it, then pass it to the acceptor, then handle the resulting Transport.

**Comment by jbr** [comment]
This is fixed on main and should be ready for a release within the next few days
